### DISCUSSION TITLE
Minecart / Boat / Entity Showers

### DISCUSSION THREAD

**Discussion by rihlsul** [GriefPrevention/GriefPrevention#716] [Ideas]
One of the many ways we've discovered that is not stopped by GriefPrevention that I don't think is a bug, so filing this as a Feature Request:

An enterprising player can build an ice-bottomed water slide near the edge of a claim, and any entities placed into the water will simply slide into the claimed property.  For example, dispensers filled with Minecarts, boats, armor stands... You end up with quite the cleanup job.

This one, I'm less sure how to prevent.  The entity spawning in the above scenario is done my machinery, which was placed outside a claim.

**Comment by rihlsul** [comment]
Note: Same strategy can also fill a claim with quite an array of items.

**Comment by ShaneBeee** [comment]
TumpIt and build a wall 😉
